### DISCUSSION TITLE
GitHub spoiler syntax

### DISCUSSION THREAD

**Discussion by AustenLamacraft** [micromark/micromark#112] [Q&A] [answered]
I'd like to display GitHub spoiler syntax correctly

<details>
  <summary>Spoiler warning</summary>
  
  It doesn't work
  
</details>

```markdown
<details>
  <summary>Spoiler warning</summary>
  
  It doesn't work
  
</details>
```

I've tried the gfm extension with no joy. Any ideas?

**Comment by ChristianMurphy** [comment, marked as answer]
The content shown is HTML, not CommonMark or GFM per se.
To enable HTML, you may want to look into the [allowdangeroushtml option](https://github.com/micromark/micromark#optionsallowdangeroushtml) (though be aware of the [security implications](https://github.com/micromark/micromark#security))
